query-connector/e2e/alternate_queries.spec.ts

```diff
@@ -0,0 +1,81 @@
+import { test, expect } from "@playwright/test";
```

Comment:
this whole page is just a copy-paste from the existing file to help parallelize the playwright runners